### DISCUSSION TITLE
Update available

### DISCUSSION THREAD

**Discussion by ThiaJay** [OpenRCT2/OpenRCT2#18278] [Q&A]
Loaded up OpenRCT 2 today and I'm told that there is an update available. I'm using the developer build 0.4.2-3, I click the button and don't see any more updates?

**Comment by J-PIE-314** [comment]
You can report bugs at the following link: [bug report](https://github.com/OpenRCT2/OpenRCT2/issues/new?assignees=&labels=bug&template=bug_report.yaml)
